### ISSUE TITLE
Cannot choose page to be visible in headful mode

### ISSUE THREAD

**Issue by jperl** [microsoft/playwright#657]
Use case: in headful mode when a test runs against multiple pages (like a chat application with two chat pages open), we want to make the page being actioned on visible on the display. This is because we run the tests in headful mode, and record the tests with ffmpeg so we can watch the test later. If we cannot bring that page to the front, the actions might be hidden behind another page.

For Puppeteer we used [page.bringToFront](https://github.com/puppeteer/puppeteer/blob/v2.0.0/docs/api.md#pagebringtofront).

**Comment by pavelfeldman** [comment]
Could you rephrase it in a form of a bug? Something like "Can't interact with the page containing video in such as such way"? We might choose to create browser-per-tab in headful mode and disable occlusion instead of forcing activation and messing with the focus, etc.

**Comment by jperl** [comment]
Thank you for your response :pray:. I tried to rephrase and clarify the use case. Is it more clear now?

**Comment by ihor-nikiforov-scout** [comment]
I need a feature to switch between pages as well so I can interact with 2 different users(2 incognito contexts) with the system under test.

page.bringToFront method would fix the issue.

**Comment by pavelfeldman** [comment]
If that is a part of the record / replay debugging, could you use CDP for it?

**Comment by jperl** [comment]
This is no longer important for our use case since we are using screencast

**Comment by pavelfeldman** [comment]
@ihor-nikiforov-scout could you share why you want to 'switch' between the pages? Both pages would think they are focused and you can interact with them simultaneously using respective page objects.

**Comment by ihor-nikiforov-scout** [comment]
@pavelfeldman the idea is to see what is happening in not headless mode.

**Comment by sublimator** [comment]
@pavelfeldman
We use puppeteer for testing an extension which performs actions such as starting/stopping micropayments based on the "active" tab. Being able to simulate a user switch tabs is useful. 
With playwright this is not possible?
